requirements.txt

```diff
@@ -1,4 +1,5 @@
-jupyterlab==3.0.3
+jupyterlab==2.2.7
```

Comment:
Why did we change the lab version?

Reply:
Resolving a dependency conflict between this version of jupyter-lab and the current version of the API examples that would cause `pip install -r requirements.txt` to fail